### PR TITLE
Added missing bridge days for Russia for 2023

### PR DESCRIPTION
**Pull request by leeloo-1337**
## Proposed change

Added missing bridge days for Russia for 2023. Determined yearly by the government: https://www.consultant.ru/document/cons_doc_LAW_425407/

## Type of change

- [ ] New country/market holidays support (thank you!)
- [x] Supported country/market holidays update (calendar discrepancy fix, localization)
- [ ] Existing code/documentation/test/process quality improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency upgrade (version update)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new python-holidays functionality in general)

## Checklist

- [x] I've followed the [contributing guidelines][contributing-guidelines]
- [x] This PR is filed against `beta` branch of the repository
- [x] This PR doesn't contain any merge conflicts and has clean commit history
- [x] The code style looks good: `make pre-commit`
- [x] All tests pass locally: `make test`, `make tox` (we strongly encourage adding tests to your code)
- [ ] The related [documentation][docs] has been added/updated (check off the box for free if no updates is required)

[contributing-guidelines]: https://github.com/dr-prodigy/python-holidays/blob/beta/CONTRIBUTING.rst
[docs]: https://github.com/dr-prodigy/python-holidays/tree/beta/docs/source

### CODE REVIEW

File: holidays/countries/russia.py [hunk 1]
```diff
@@ -23,6 +23,11 @@ class Russia(HolidayBase):
     country = "RU"
     default_language = "ru"
 
+    special_holidays = {
+        2023: ((FEB, 24, "День защитника Отечества"),), # Bridge day for 1/Jan, src: https://www.consultant.ru/document/cons_doc_LAW_425407/
+        2023: ((MAY, 8, "День Победы"),), # Bridge day for 8/Jan, src: https://www.consultant.ru/document/cons_doc_LAW_425407/
```
Comment (arkid15r) [inline holidays/countries/russia.py:28]: These keys are identical and the latter key overwrites former one.
This is why we emphasize on importance of [adding tests](https://github.com/dr-prodigy/python-holidays/blob/beta/tests/countries/test_russia.py).

File: holidays/countries/russia.py [hunk 2]
```diff
@@ -23,6 +23,11 @@ class Russia(HolidayBase):
     country = "RU"
     default_language = "ru"
 
+    special_holidays = {
+        2023: ((FEB, 24, "День защитника Отечества"),), # Bridge day for 1/Jan, src: https://www.consultant.ru/document/cons_doc_LAW_425407/
```
Comment (arkid15r) [inline holidays/countries/russia.py:27]: Could you really run the pre-commit and make sure the code style looks good? Thank you!

**Review by arkid15r — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please fix the issues and request the review again.

**Comment by arkid15r** [comment]
Some issues still need to be fixed. Could you also add tests for these holidays?

**Comment by leeloo-1337** [comment]
> Some issues still need to be fixed. Could you also add tests for these holidays?

The test ran okay on the latest one. Unless I'm doing it wrong?

File: holidays/countries/russia.py [hunk 3]
```diff
@@ -23,6 +23,14 @@ class Russia(HolidayBase):
     country = "RU"
     default_language = "ru"
 
+    special_holidays = {
+        2023: (
+            # Bridge days for 01/Jan & 08/Jan
```
Comment (arkid15r) [inline holidays/countries/russia.py:28]: Could you keep the link to the law too?

File: tests/countries/test_russia.py [hunk 4]
```diff
@@ -43,6 +43,12 @@ def test_2018(self):
             "2018-11-04",
         )
 
+    def test_2023(self):
```
Comment (arkid15r) [inline tests/countries/test_russia.py:46]: Thanks for adding the tests.

Could you keep the 2018's tests together?

File: holidays/countries/russia.py (hunk 3 above)
Comment (leeloo-1337) [reply holidays/countries/russia.py:28]: > Could you keep the link to the law too?

Done

**Review by arkid15r — COMMENTED** [review]
Thank you for addressing the comments, marking this as ready for beta.

**Pull request merged**